.github/workflows/build.yml

```diff
@@ -22,34 +22,25 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v4
 
-      - name: Install stable toolchain
+      - name: Install nightly toolchain
         uses: actions-rs/toolchain@v1
         with:
-          profile: default
-          toolchain: stable
```

Comment:
i still think we should keep both. 
stable for crates, nightly for format and unstable options.

Reply:
but the `rust-toolchain.toml` just override it

Reply:
does it support multiple toolchains?

Reply:
can we pls use `cargo +nightly-YYYY-MM-DD fmt` instead of using `rust-toolchain` and forcing the compiler of normal code to be nightly. We really only use nightly for fmt and for unstable option of report time in CI. 
(thanks for exploring this possibility tho)

imo, it's bad to force jellyfish user to be on nightly. (some downstream repo only support stable rust)